core/src/main/kotlin/fr/sncf/osrd/standalone_sim/ScheduleMetadataExtractorV2.kt

```diff
@@ -225,10 +227,26 @@ fun makeSimpleReportTrain(
             return@rdp abs(point.speed - projSpeed) * speedScaling +
                 abs(point.time - projTime) * timeScaling
         }
+
+    // Check if the scheduled points were respected
+    var scheduledPointsHonored = true
+    for (e in schedule) {
+        if (e.arrival == null) continue
+        var computed = envelopeStopWrapper.interpolateTotalTimeUS(e.pathOffset.distance.meters)
+        if (e.stopFor != null) computed -= e.stopFor.milliseconds * 1_000
```

Comment:
We could do the computation with seconds there, I don't think we need high accuracy and juggling ms with µs is confusing